apps/src/blockly/addons/cdoBlockFlyout.ts

```diff
@@ -210,7 +210,6 @@ export default class CdoBlockFlyout extends GoogleBlockly.HorizontalFlyout {
    *     area.
    * @override
    */
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
we have configured this lint rule to allow unused function parameters: https://github.com/code-dot-org/code-dot-org/blob/82cfeab481439846eebb0a60990182499c22bf51/apps/.eslintrc.js#L193